.storybook/main.js

```diff
@@ -52,6 +52,9 @@ module.exports = {
     const customAlias = {
       'react-dom': path.resolve(__dirname, '../node_modules/react-dom/profiling'),
       'scheduler/tracing': 'scheduler/tracing-profiling',
+      '@emotion/core': path.resolve(__dirname, '../node_modules/@emotion/react'),
```

Comment:
These tell webpack to resolve our the emotion dependencies to the versions used by the monorepo.

The styles previously were not being applied in storybook because webpack was resolving imports from the `@emotion` packages locally (e.g. to the bundled storybook dependencies). This is problematic because:
1. it was using a v10 version of the theming library
2. it was using the root version of the styled library

Essentially this means that our consumer (rendered by `styled` components) and provider (`Theme.Provider`) could not use the same context. The `theme` object in the consumer was empty because it is not coupled to the provider we were using!

It also kind of explains why I was seeing the following rendered:
````
// in the React component dev tools
// running with env "dev"
<ThemeProvider>
  <EmotionThemeContext.Consumer>
      {/* children here */}
  </EmotionThemeContext.Consumer>
</ThemeProvider>
````

Emotion v11 now uses the display name `EmotionThemeContext` in development. This led into checking to see if we were using multiple versions of the emotion dependencies, which we were.

Reply:
praise: nice detective work! Solid fix.